searx/settings.yml

```diff
@@ -1334,6 +1334,31 @@ engines:
     url: https://thepiratebay.org/
     timeout: 3.0
 
+  - name: pixiv
+    shortcut: pv
+    engine: pixiv
+    disabled: true
+    inactive: true
+    pixiv_image_proxies:
+      - pximg.example.org
```

Comment:
I think this line should better be removed or commented since it's not an actual working proxy url

Reply:
Oh, I guess it's related to the `settings.yml.new` auto-new-changes-adder, right?

Reply:
If I comment the `pximg.example.org` list item out the indentation of the comments are reported as error:

```
make test
TEST      [yamllint] $YAMLLINT_FILES
searx/settings.yml:1343:7: [warning] comment not indented like content (comments-indentation)
ERROR: test.yamllint exit with error (2)
make: *** [Makefile:95: test.yamllint] Fehler 2
```

Here is the config that produce error from above:

```yaml
  - name: pixiv
    shortcut: pv
    engine: pixiv
    disabled: true
    inactive: true
    pixiv_image_proxies:
      # - pximg.example.org
      # A proxy is required to load the images. Hosting an image proxy server
      # for Pixiv:
      #    --> https://codeberg.org/VnPower/PixivFE/wiki/Hosting-an-image-proxy-server-for-Pixiv
      # Proxies from public instances.  Ask the public instances owners if they
      # agree to receive traffic from SearXNG!
      #    --> https://codeberg.org/VnPower/PixivFE#instances
      #    --> https://github.com/searxng/searxng/pull/3192#issuecomment-1941095047
      # image proxy of https://www.pixiv.net
      # - https://i.pixiv.cat
      # image proxy of https://www.pixiv.pics
      # - https://pximg.cocomi.eu.org
      # image proxy of https://pixivfe.exozy.me
      # - https://pximg.exozy.me
      # image proxy of https://pixivfe.ducks.party
      # - https://pixiv.ducks.party
      # image proxy of https://pixiv.perennialte.ch
      # - https://pximg.perennialte.ch
```